.travis/prepare.sh

```diff
@@ -1,14 +1,49 @@
 #!/bin/bash
 
-cd ../..
-mkdir cilium
-mv pchaigno/cilium cilium/cilium
-cd cilium/cilium
-
-curl https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-sudo apt-add-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main"
-sudo apt-get -qq update
-sudo apt-get install -y clang-10 llvm-10
+CLANG_VERSION=10.0.0
+
+function setup_env() {
+case `uname -m` in
+  'x86_64' )
+    CLANG_DIR="clang+llvm-$CLANG_VERSION-x86_64-linux-gnu-ubuntu-18.04"
+    ;;
+  'aarch64' )
+    CLANG_DIR="clang+llvm-$CLANG_VERSION-aarch64-linux-gnu"
+    ;;
+esac
+}
+
+function install_clang() {
+  CLANG_FILE="$CLANG_DIR.tar.xz"
+  CLANG_FILE_SIG="$CLANG_FILE.sig"
+
+  CLANG_URL="https://github.com/llvm/llvm-project/releases/download/llvmorg-$CLANG_VERSION/$CLANG_FILE"
+  wget -nv $CLANG_URL
+  CLANG_URL="https://github.com/llvm/llvm-project/releases/download/llvmorg-$CLANG_VERSION/$CLANG_FILE_SIG"
+  wget -nv $CLANG_URL
+
+  wget -nv https://releases.llvm.org/3.7.0/hans-gpg-key.asc
```

Comment:
For consistency we might want to use the public key file for the same release:

```suggestion
  wget -nv https://releases.llvm.org/$CLANG_VERSION/hans-gpg-key.asc
```

Reply:
Good comments, Thanks
I have updated this patch.